### PR TITLE
Register member

### PR DESCRIPTION
**Pull request by ilzeLourens**

### CODE REVIEW

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 1]
```diff
@@ -441,7 +443,7 @@ export default class search extends Component {
           <div className={this.state.ImageTaken ? "image-container" : "hidden"}>
             <div className="centre-label">
               <div className={!this.state.ImageTaken ? "hidden" : "label-score photo-top-label"}>
-                <b>Score Captured</b>
+                Score Captured
```
Comment (laurent-mbuyu) [inline skietbaan_fe/src/components/ScoreCapture.js]: Put it in an html element

Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: what element ?

Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: its in a div

Comment (laurent-mbuyu) [reply skietbaan_fe/src/components/ScoreCapture.js]: p element

Comment (ilzeLourens) [reply skietbaan_fe/src/components/ScoreCapture.js]: I'm sorry, but I dont understand why

Comment (laurent-mbuyu) [reply skietbaan_fe/src/components/ScoreCapture.js]: The div tag defines the division of a section in an html just like the external wall of your house.
The p tag and other tags are like the walls in your house.
With such, you can independently make modifications of the p tag independently of the div element

File: skietbaan_fe/src/components/ScoreCapture.js [hunk 2]
```diff
@@ -382,22 +384,22 @@ export default class search extends Component {
             <div className={(this.state.showCamera && !this.state.ImageTaken) 
               || this.state.ImageTaken || !this.state.scoreEntered ? "hidden" : "submit-button-elements"}>
               <div className="button-hover ">
-                <img src={require('../components/assets/submitScore.png')} onClick={() => this.GetLocation()}
+                <img src={require('../components/assets/graySubmit.png')} onClick={() => this.GetLocation()}
```
Comment (FredJEhlers) [inline skietbaan_fe/src/components/ScoreCapture.js]: these should be pulled into constants object please

**Pull request merged**